ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/permissioning/NodeWhitelistController.java

```diff
@@ -109,6 +107,16 @@ public NodesWhitelistResultType result() {
     }
   }
 
+  public boolean contains(final Peer node) {
+    return (!nodeWhitelistSet || (nodesWhitelist.contains(node)));
+  }
+
+  public boolean contains(
+      final BytesValue nodeId, final String host, final int udpPort, final int tcpPort) {
+    return (!nodeWhitelistSet
```

Comment:
This seems like dupe.  Could it be replaced with a call to `contains(new DefaultPeer(nodeId, host, udpPort, tcpPort))` ?